ibis/tests/expr/test_value_exprs.py

```diff
@@ -1662,3 +1641,16 @@ def test_where_output_shape():
     expr = ibis.literal(True).ifelse(t.a, -t.a)
     assert isinstance(expr, ir.IntegerColumn)
     assert isinstance(expr, ir.ColumnExpr)
+
+
+def test_quantile_shape():
+    t = ibis.table([("a", "float64")])
+
+    b1 = t.a.quantile(0.25).name("br2")
+    assert isinstance(b1, ir.Scalar)
+
+    projs = [b1]
+    expr = t.projection(projs)
```

Comment:
```suggestion
    expr = t.select(projs)
```
Try to avoid using `projection`, we should move away from it as the conventional API to select/compute columns.

Reply:
Updated.